arm9/src/unk_02022450.c

```diff
@@ -0,0 +1,43 @@
+#include "global.h"
+#include "FS_file.h"
+#include "OS_terminate_proc.h"
+#include "CARD_rom.h"
+#include "MI_memory.h"
+
+#include "unk_02022450.h"
+
+static const char string_saying_rom[] = "rom";
+
+THUMB_FUNC void FUN_02022450 () {
+    if (!FS_IsAvailable()) {
+        OS_Terminate();
+    }
+    else {
+        u32 *r1 = (u32*)HW_MAIN_MEM_SHARED;
+        if (!*(u32*)0x027FF00C) {
```

Comment:
a constant can probably be created for this, it seems to be some sort of protection value located in shared work, it's equal to HW_MAIN_MEM_SHARED[0xC]